<?xml version="1.0" encoding="utf-8"?>
<report
	progress="">
	<activity
		result="Failed"
		duration="1.2m"
		durationms="72870"
		type="root"
		rid="48335dad1be8870"
		totalerrorcount="1"
		totalwarningcount="0"
		totalsuccesstestcasecount="0"
		totalfailedtestcasecount="0"
		totalblockedtestcasecount="0"
		childsuccesscount="0"
		childfailedcount="0"
		childblockedcount="0"
		totalsuccesscount="0"
		totalfailedcount="0"
		totalblockedcount="0"
		totalmaintenancemodecount="0"
		user="Fahad Shah"
		rxversion="8.0.0.0"
		host="HUAWEI LUA-U22 (Unknown)
WLAN - 192.168.3.11"
		osversion="-"
		runtimeversion="-"
		procarch="-"
		oslanguage="-"
		dtnlanguage="-"
		screenresolution="-"
		timestamp="3/6/2018 6:46:02 PM"
		timestampiso="2018-03-06T13:46:02"
		endtime="3/6/2018 6:47:15 PM"
		timeoutfactor="1"
		module="MarkAttendance">
		<detail>
			MarkAttendance
		</detail>
		<varbindings>
		</varbindings>
		<item
			timeRelativeToTestSuiteStartTime="01:12.869"
			timeRelativeToTestModuleStartTime="01:12.873"
			timeWallClock="18:47:15"
			time="01:12.869"
			level="Error"
			category="Module">
			<message>
				The test run has been aborted by the user.
			</message>
			<metainfo
				loglvl="Error">
			</metainfo>
		</item>
	</activity>
</report>
